libraries/botframework-connector/src/auth/parameterizedBotFrameworkAuthentication.ts

```diff
@@ -304,6 +312,14 @@ export class ParameterizedBotFrameworkAuthentication extends BotFrameworkAuthent
         authHeader: string,
         channelId: string
     ): Promise<ClaimsIdentity> {
+        // Add allowed token issuers from configuration (if present)
+        if (this.authConfiguration.validTokenIssuers?.length > 0) {
+            ToBotFromBotOrEmulatorTokenValidationParameters.issuer = [
```

Comment:
It seems like this list will continue to grow as long as a bot runs since we're pushing entries into a static array. Is that a concern?

Reply:
We updated how the issuers were assigned, we defined a shallow clone object assigning new `issuers` without altering the `ToBotFromBotOrEmulatorTokenValidationParameters` constant object.